go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/googollee/go-socket.io
 go 1.13
 
 require (
-	github.com/googollee/go-engine.io v1.4.3-0.20200220091802-9b2ab104b298
+	github.com/googollee/go-engine.io v1.4.2
```

Comment:
I feel bad for making so many comments on this small (but very great!) pull request. It looks like this downgrades the dependency here. Is that intentional?

Reply:
No, but there isn't a `v1.4.3` tag on the go-engine.io repo. Looks like `v1.4.3` tag for this repo was [just created yesterday](https://github.com/googollee/go-socket.io/releases/tag/v1.4.3). I suspect that `go mod tidy` is just picking up the latest tag.

If the goal is to keep the versions in sync, I _think_ the thing to do is tag both a `v1.4.3` and `v1.4.4` release on go-engine.io repo. Those could be both point to the same commit.

Once that's done, bump the version in this PR, merge it, then tag a `v1.4.4` of go-socket.io.